<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editSeance</class>
 <widget class="QDialog" name="editSeance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 48, 48);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="labelimg">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(212, 212, 212);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="styleSheet">
      <string notr="true">QCalendarWidget QToolButton {

width:150px;
color:white;
font-size:20px;
icon-size:56px,56px;
background-color: rgb(0, 0, 0);
}

QCalendarWidget QWidget{
alternate-background-color:rgb(128,128,128)
}

QCalendarWidget QAbstractItemView:enabled{
font-size:18px;
color:rgb(180,180,180);
background-color:black;
selection-background-color:rgb(64,64,64);
selection-color:rgb(0,255,0);
}
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"> QGroupBox {

	color: rgb(0, 0, 0);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #D4D4D4, stop: 1 #D4D4D4);
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #C8C8C8, stop: 1 #E6E6E6);
 }</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="nombreSeances_4">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Seances dans la journee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nombreSeances_2">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Nombre de seances</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dateDep">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Date Depart</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="heureDep">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Heure Depart</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dateAr">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Date arrivee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="heureAr">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Heure arrivee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="type_2">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="seancesJournne">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="nombreSeances">
              <property name="styleSheet">
               <string notr="true">QSpinBox {
background-color: rgb(100,100, 100);
padding-right: 10px; /* Crée un espace pour les flèches */
border-width: 3;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateDepart">
              <property name="styleSheet">
               <string notr="true">QDateEdit {
background-color: rgb(100,100, 100);
padding-right: 10px; /* Crée un espace pour les flèches */
border-width: 3;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="heureDepart">
              <property name="styleSheet">
               <string notr="true">QTimeEdit {
background-color: rgb(100,100, 100);
padding-right: 10px; /* Crée un espace pour les flèches */
border-width: 3;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateArrivee">
              <property name="styleSheet">
               <string notr="true">QDateEdit {
background-color: rgb(100,100, 100);
padding-right: 10px; /* Crée un espace pour les flèches */
border-width: 3;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="heureArrivee">
              <property name="styleSheet">
               <string notr="true">QTimeEdit {
background-color: rgb(100,100, 100);
padding-right: 10px; /* Crée un espace pour les flèches */
border-width: 3;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="type">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnMod">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	heigth:40px;
	width:150px;
	color:black;
	font-size:20px;
	icon-size:56px,56px;
	color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>labelimg</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
